src/app/malaria-101/activity-1/activity-1-2.component.html

```diff
@@ -0,0 +1,21 @@
+
+<img *ngFor="let cards of createRange(5); let idx = index" class="arrow-{{idx+1}}" src="../../../../assets/img/red-up-arrow.png">
+
+<div *ngFor="let cards of createRange(6); let idx = index" [id]="idx + 1" class="drop-{{idx+1}}" (drop)="drop($event)" (dragover)="allowDrop($event)">
+</div>
+<div *ngFor="let card of labelsArr; let idx = index" class="text-{{idx+1}}">
+    <p>{{card}}</p>
+</div>
+<div *ngFor="let card of solnArr; let idx = index" class="drag-{{idx+1}}" (drop)="drop($event)" (dragover)="allowDrop($event)">
+    <img class="pieces" src="../../../../assets/img/{{card}}" draggable="true" (dragstart)="drag($event)" id="drag{{idx+1}}">    
+</div>
+
+<div class="row">
+    <button class="arrow-dnd-red col-lg-5 col-md-offset-2">
```

Comment:
what's the difference between `arrow-red` & `arrow-dnd-red`?
also, wouldn't a directive be better for the prev & next buttons? the same button should be used across all activities anyway